plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/tools/configuration/ConfigurationImportData.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jkiss.dbeaver.tools.configuration;
+
+import org.jkiss.code.NotNull;
+
+public class ConfigurationImportData {
+
+    private final String filePath;
+
+    public ConfigurationImportData(@NotNull String filePath) {
+        this.filePath = filePath;
+    }
+
+    public String getFilePath() {
+        return filePath;
+    }
+}
```

Comment:
What's the point of having two identical classes with different names? Let's call it `ConfigurationTransferData` and delete one of them.

Reply:
There is a possibility of further functionality in the future this will require this classes later.